endpoint.cpp

```diff
@@ -371,14 +371,28 @@ int UartEndpoint::open(const char *path, speed_t baudrate)
 
     tc.c_iflag &= ~(IGNBRK | BRKINT | ICRNL | INLCR | PARMRK | INPCK | ISTRIP | IXON);
     tc.c_oflag &= ~(OCRNL | ONLCR | ONLRET | ONOCR | OFILL | OPOST);
-    tc.c_lflag &= ~(ECHO | ECHOE | ECHOK | ECHOCTL | ECHOKE | ECHONL | ICANON | IEXTEN | ISIG | TOSTOP);
-    tc.c_cflag &= ~(CSIZE | PARENB | CBAUD | CRTSCTS);
+
+    tc.c_lflag &= ~(ECHO | ECHOE | ECHOK | ECHOCTL | ECHOKE | ECHONL | ICANON | IEXTEN | ISIG);
+
+    /* never send SIGTTOU*/
+    tc.c_lflag &= ~(TOSTOP);
+
+    /* disable flow control */
```

Comment:
we currently don't support flow control (and we didn't before...). We need to properly enable it

> 			PX4_WARN("hardware flow control not supported");

for the hardwares we currently support, PX4 would print this message, because flow control is disabled on the stm32 side